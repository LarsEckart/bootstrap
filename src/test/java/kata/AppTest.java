package kata;

import static org.assertj.core.api.Assertions.assertThat;

import org.junit.jupiter.api.DisplayNameGeneration;
import org.junit.jupiter.api.DisplayNameGenerator;
import org.junit.jupiter.api.Test;

@DisplayNameGeneration(DisplayNameGenerator.ReplaceUnderscores.class)
class AppTest {

  @Test
  void my_first_test() {
    assertThat("4" + "2").isEqualTo("42");
  }
}
