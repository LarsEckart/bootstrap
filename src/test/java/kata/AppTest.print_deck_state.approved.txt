|XX| |XX| |XX| 
|XX| |12| |12| 
|XX| |XX| |XX| 
|XX| |XX| |XX| 

|XX| |XX| |XX| 
|XX| |11| |11| 
|XX| |XX| |XX| 
|XX| |XX| |XX| 

